laser_slam_ros/src/laser_slam_worker.cpp

```diff
@@ -1,3 +1,4 @@
+#include "laser_slam/benchmarker.hpp"
```

Comment:
@dotPiano , starting with the "corresponding" is not just convention but a very good thing: This way the includes in the corresponding header are best tested.